src/cursor-doc/model.ts

```diff
@@ -155,12 +155,12 @@ export class LineInputModel implements EditableModel {
           }
           return x;
         })
-        .filter((x) => x !== null)
+        .filter((x) => x !== null) as number[]
```

Comment:
We know from the code this is the correct type so we can just assert it. TypeScript wasn't smart enough to know that filtering out nulls would make a `(number|null)[]` into a `number[]`

Reply:
Cool. Thanks for mention I learn from them. ❤️